.circleci/e2e-job-ignore-patterns.txt

```diff
@@ -6,5 +6,15 @@
 .log
 .pdf
 .gitignore
-.java
-.kt
\ No newline at end of file
+Test.java
+.kt
+.tsx
+.git
+yarn.lock
+config.yml
+pr-halt-e2e.sh
+e2e-test-users.sh
```

Comment:
Including every script name in here I'm not sure makes sense, maybe this was just for iteration. Would we not want to rerun CI when we change the CI settings? Seems like we should..

Reply:
Right. I can't remember why I did this (worked on PR for 3 days) I'm going to remove `.sh` files and `config.yml`.